ci/docker_ci.sh

```diff
@@ -12,12 +12,12 @@ ENVOY_DOCKER_IMAGE_DIRECTORY="${ENVOY_DOCKER_IMAGE_DIRECTORY:-${BUILD_STAGINGDIR
 
 # Setting environments for buildx tools
 config_env() {
-  # Qemu configurations
-  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+  # Install QEMU emulators
+  docker run --rm --privileged tonistiigi/binfmt --install all
```

Comment:
i need to understand better what is changed here 

is `tonistiigi/binfmt --install all` a dropin replacement for `tonistiigi/binfmt --install all` ?

Reply:
yes

Reply:
wfm